tensorboard/pip_package/setup.py

```diff
@@ -62,8 +62,6 @@ def get_readme():
             "tf_projector_plugin/projector_binary.js",
         ],
     },
-    # Disallow python 3.0 and 3.1 which lack a 'futures' module (see above).
```

Comment:
Optional but maybe kill the `futures` and `wheel` deps that were py2-only too?

https://github.com/tensorflow/tensorboard/blob/b53c723980a5429d81e0443772b41a26334ea288/tensorboard/pip_package/requirements.txt#L19-L20

Reply:
Done.

Reply:
It may be nice to re-add python_requires with `python_requires=">=3.6"`, it will block folks with older version Python version from installing the library through pip https://packaging.python.org/guides/dropping-older-python-versions/

Reply:
Adding one in #5083.